src/zenml/zen_server/deploy/helm/templates/hpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2
```

Comment:
What implications does this have on the supported Kuberentes cluster versions ? Should this be done with an if/else depending on that kube API version ?

Reply:
With this change, we would stop supporting clusters with versions starting 1.23 and older. I think it's reasonable to no longer support that since 1.23 has reached end of support almost 6 months back and most cloud providers use 1.27 as the default cluster version.